# type = -1 is waiting for image
# type =  0 is not have object
# type >= 1 is have a object
int8 type
float64 cx
float64 cy
float64 area